book/src/basics/installation.md

```diff
@@ -0,0 +1,17 @@
+# Installation
```

Comment:
This section is a good place to document the relevant portions of the Prometheus and Grafana installations like I did in the README. Users shouldn't be left to figure that out by themselves.

Firewall configuration should also appear here. You can take it from the README as well.

Reply:
How does 96d714a look?